src/algorithms/standard/peakdetection.h

```diff
@@ -55,6 +56,7 @@ class PeakDetection : public Algorithm {
     declareParameter("threshold", "peaks below this given threshold are not output", "(-inf,inf)", -1e6);
     declareParameter("orderBy", "the ordering type of the output peaks (ascending by position or descending by value)", "{position,amplitude}", "position");
     declareParameter("interpolate", "boolean flag to enable interpolation", "{true,false}", true);
+    declareParameter("minPeakDistance", "minimum distance between consecutive peaks (0 to bypass this feature)", "[0,inf)", 0.0);
```

Comment:
Maybe it's worth to mention in the doc (here or in the description DOC string, that minPeakDistance relies on picking up the predominant peaks)